src/App.js

```diff
@@ -1,16 +1,20 @@
+import { render } from '@testing-library/react';
 import './App.css';
 
+const multiplier = 0;
+
+
 function App() {
-  
-  chickenClick = () => {
-    console.log('this is:', this);
-  };
+
+  function chickClick() {
+    multiplier = multiplier + 1;
+  }
 
   return (
     <div className="App" >
       <header className="App-header">
         <h1>Farm Idle Game</h1>
-        <button type="button" onClick={chickenClick}>Click Me!</button>
+        <button type="button" onClick={chickClick}>Click Me!, {multiplier}</button>
```

Comment:
You might wish to display this counter in a separate element than the button itself?